sc/social/like/upgrades/v3046/__init__.py

```diff
@@ -5,6 +5,15 @@
 
 import transaction
 
+try:
+    # this exception must be resubmitted:
+    from ZODB.POSException import ConflictError
+except ImportError:
```

Comment:
if we don't require it, better to remove it.

Reply:
I think it's best to add `ZODB` to setup.py